### ISSUE TITLE
GraphSage-Encoder use_id=true,use_feature=true，embedding是怎么更新的呢？

### ISSUE THREAD

**Issue by dylanwinty** [alibaba/euler#144]
**base_layers.py中：embedding 的call**
 def call(self, inputs):
    shape = inputs.shape
    inputs = tf.reshape(inputs,[-1])
    output_shape = shape.concatenate(self.dim) 
    output_shape = [d if d is not None else -1 for d in output_shape.as_list()]  #//tensorshape->[,,] list 
    return tf.reshape(tf.nn.embedding_lookup(self.embeddings, inputs),output_shape)

GraphSage-ShallowEncoder encoder中
 def call(self, inputs):
    input_shape = inputs.shape
    inputs = tf.reshape(inputs, [-1])
    embeddings = []

    if self.use_id:
      embeddings.append(self.embedding(inputs))

    if self.use_feature:
      features = sample.get_dense_feature(inputs)
      features = tf.concat(features, -1)
      if self.combiner == 'add':
        features = self.dense(features)
      embeddings.append(features)
只看到去读self.embedding的代码。没有看到每个id的embedding何时更新啊。但是ScaleableGraphSage encoder就有去更新embedding的代码。

那这个id的embedding是何时更新的呢？
